jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/GradleLayerConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,131 @@
+package com.google.cloud.tools.jib.gradle;
+
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class GradleLayerConfigurationIntegrationTest {
+
+  @ClassRule
+  public static final TestProject multiTestProject = new TestProject("all-local-multi-service");
+
+  @Test
+  public void testGradleLayerConfiguration_multiModule() throws IOException {
+    multiTestProject.build(":complex-service:jibBuildTar");
```

Comment:
Is it possible to run Jib on the root project, or it is recommend to call Jib on sub-modules this way?

Reply:
It kind of is just doing what I want it to. I guess I could make it one test and check everything, is that you what you were expecting?

Reply:
Not really expecting it. If this does what you want to do, that's good. I was just wondering if `all-local-multi-module` is in a structure found in practice and what is the correct way to set up and call Jib.

Reply:
Yeah, I think it's somewhat realistic. It might be unnecessary to put `simple-service` in there. But to execute things one at a time, this is the way I would do it.